Source/Core/Common/Timer.cpp

```diff
@@ -261,4 +261,13 @@ double Timer::GetDoubleTime()
 	return TmpTime;
 }
 
+std::string Timer::GetDateTimeFormatted(double time)
+{
+	time_t seconds = (time_t)time + (38 * 365 * 24 * 60 * 60);
```

Comment:
I didn't mean changing/removing it, but making it a constant (in both places, preferably). That way you can also see where the constant is used and find the connection.